introduction.md

```diff
@@ -1,3 +1,7 @@
 Hi. My name is Maahd. 
 I'm working as a software developer at a small startup in Germany. 
 My favorite CodeSchool path is iOS. 
+My deadly skills are:
+* Pyhton
+* Java
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?